src/store/provider.js

```diff
@@ -37,8 +37,10 @@ export const state = () => ({
 })
 
 export const getters = {
-  getProviderName: (state, getters, rootState) => (providerCode) => {
-    const mediaType = rootState.search.mediaType
+  getProviderName: (state) => (providerCode) => {
+    const searchStore = useSearchStore()
+    const mediaType =
+      searchStore.searchType === ALL_MEDIA ? IMAGE : searchStore.searchType
```

Comment:
Agree. I think this might get fixed when we convert the provider store to Pinia. It actually needs to be refactored just like the other media files instead of hard-coding `audio` and `image`.